cassandra_init_scripts/keyspace.cql

```diff
@@ -11,3 +11,31 @@ CREATE TABLE IF NOT EXISTS stream_data (
     payload blob,
     PRIMARY KEY ((id, partition), ts, sequence_no, publisher_id, msg_chain_id)
 );
+
+
+-- ### NEW SCHEMA ### --
+create table bucket
+(
+    stream_id   text,
+    partition   int,
+    date_create timestamp,
+    id          timeuuid,
+    records     int,
+    size        int,
+    primary key ((stream_id, partition), date_create)
+) with clustering order by (date_create desc);
+
+create table stream_data_new
+(
+    stream_id    text,
+    partition    int,
+    bucket_id    timeuuid,
+    ts           timestamp,
+    sequence_no  int,
```

Comment:
- This `sequence_no` used to be `bigint` and is now `int`. Is this on purpose?
- Should we have `bucket_id` just be `timestamp` as they related to `ts` fields that are timestamps?

Reply:
1. About `sequence_no`
`bigint` = 64-bit signed long
`int` = 32-bit signed long
I don't think that we need 64-bit for `sequence_no`, we can save a bit disk space.

2. I'd like to keep `timeuuid` for `bucket_id`, because we get nice uuids `6cf7aa11-a56d-11ea-aa93-f14bd4097ea1`.

Reply:
<img width="1094" alt="Screenshot 2020-06-04 at 12 11 46" src="https://user-images.githubusercontent.com/7105861/83738206-a98ccb00-a65c-11ea-9917-3dc24cf25bc7.png">